contract/src/nft_core.rs

```diff
@@ -1,5 +1,6 @@
 use crate::*;
 use near_sdk::{ext_contract, Gas, PromiseResult};
+#[allow(unused_variables)]
```

Comment:
This is unusual for contract development. Are you sure you need it?

Reply:
Thanks, already removed.